### ISSUE TITLE
Cannot input unicode character with `Ctrl+Shift+U` in the Editor

### ISSUE THREAD

**Issue by sphh** [spyder-ide/spyder#4126]
## Description of your problem

**What steps will reproduce the problem?**

1. Write α with CTRL+SHIFT+u03b1 in the editor window.

**What is the expected output? What do you see instead?**

I expect an α to be displayed, but that does not happen.
I can select the character in the `charmap` program and copy and paste it into the editor.

**Please provide any additional information below**

I came across this issue, because I use [`autohotkey`](http://autokey.googlecode.com/) to generate my Greek letters (might be a solution to bug #879), where I use the following script to insert α:
```
keyboard.send_keys('<ctrl>+<shift>+u03b1 ')
```
This works in Firefox, Thunderbird, gnome-terminal, xed, ... but in Spyder I get `03b1 ` and not α.

## Versions and main components

* Spyder Version: 3.1.2 
* Python Version: 3.5.2 64bits
* Operating system: Linux Mint 18.1

## Dependencies

Please go to the menu entry `Help > Optional Dependencies` (or
`Help > Dependencies`), press the button `Copy to clipboard`
and paste the contents below:

jedi >=0.8.1     :  0.9.0 (OK)
matplotlib >=1.0 :  2.0.0 (OK)
nbconvert >=4.0  :  5.1.1 (OK)
numpy >=1.7      :  1.12.0 (OK)
pandas >=0.13.1  :  0.19.2 (OK)
pep8 >=0.6       :  1.7.0 (OK)
psutil >=0.3     :  5.0.1 (OK)
pyflakes >=0.6.0 :  1.5.0 (OK)
pygments >=2.0   :  2.2.0 (OK)
pylint >=0.25    :  1.6.5 (OK)
qtconsole >=4.2.0:  4.2.1 (OK)
rope >=0.9.4     :  0.9.4-1 (OK)
sphinx >=0.6.6   :  1.5.2 (OK)
sympy >=0.7.3    :  1.0 (OK)

**Comment by mvillion** [comment]
I have the same problem. Ctrl-Shift-u is used to input unicode in Ubuntu.
Ctrl-Shift-u b5 gives: µ
Ctrl-Shift-u b0 gives: °
It works with my PyQt5 and PySide6 apps. It could also work in Spyder.
Ctrl-Shift-U is used for Toggle Uppercase in Spyder.
Maybe changing this short-key could restore Ubuntu unicode functionality?

**Comment by ccordoba12** [comment]
Thanks for letting us know about that @mvillion! We'll change that shortcut in a future version so that `Ctrl+Shift+U` can be used to easily enter unicode characters in the editor.

**Comment by ccordoba12** [comment]
@jsbautista, let's change the toggle uppercase shortcut to `Alt+Shift+U` to fix this issue.

**Comment by sphh** [comment]
Thanks!

Just an observation:
`Alt+Shift+U` → Uppercase
`Ctrl+U` → Lowercase

Wouldn't it be more logic, to have `Alt+U` to switch to lowercase (but honestly, I don't know, if using the `Alt` modifier for a command is possible at all …)?

**Comment by ccordoba12** [comment]
Good point, thanks for bringing it up @sphh! I checked and that shortcut is not taken in Editor.

@jsbautista, please implement @sphh's suggestion.
